### ISSUE TITLE
Can I solve 6d diffusion-like equation with FiPy?

### ISSUE THREAD

**Issue by ovgeorge** [usnistgov/fipy#827]
I have a Vlasov equation

![image](https://user-images.githubusercontent.com/12253907/140609678-1f140f8b-9115-4cdb-ae7b-c197ab8cb29f.png)

with f \in R^6. Can I use FiPy to solve it?

**Comment by wd15** [comment]
I don't think this will work in FiPy. Is there a way to write equations in higher dimensions in terms of multiple equations in lower dimensions?
